gcp/shared/utils/crypto_utils.py

```
@@ -0,0 +1,52 @@
+Fernet.generate_key(). Store the raw string in Secret Manager.
+"""
+
+import os
+
+from cryptography.fernet import Fernet
+
+_key = os.environ.get("PHONE_ENCRYPTION_KEY")
+if not _key:
+    raise RuntimeError(
+        "PHONE_ENCRYPTION_KEY is not set -- check Secret Manager mount in functions.yaml"
+    )
```

Ruff (0.15.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)